extensions/pkg/controller/shoot_test.go

```diff
@@ -19,10 +19,11 @@ var _ = Describe("Shoot", func() {
 	lastOperationReconcile := gardencorev1beta1.LastOperation{Type: gardencorev1beta1.LastOperationTypeReconcile}
 
 	cidr := "10.250.0.0/19"
+	cidrV6 := "2001:db8::/64"
 
-	DescribeTable("#GetPodNetwork",
-		func(cluster *Cluster, cidr string) {
-			Expect(GetPodNetwork(cluster)).To(Equal(cidr))
+	DescribeTable("#Get*Network",
```

Comment:
DescribeTable("#GetNetwork",
```suggestion
	DescribeTable("#GetNetwork",
```